ts/main.ts

```diff
@@ -287,6 +287,18 @@ async function fetchData(usrInput: string): Promise<void> {
       monsterImgContainer.appendChild(monsterImg);
       $selectedSq.appendChild(monsterImgContainer);
       $selectedSq.classList.add('bg-green-500', 'correct');
+      $selectedSq.classList.remove('hover:bg-yellow-100');
+      numCorrect++;
+      guesses++;
+      completedSquares++;
+      setStatsText();
+      clearSelectedSqResetModal($selectedSq);
+    } else {
+      guesses++;
+      completedSquares++;
+      setStatsText();
+      $selectedSq.classList.add('bg-red-500', 'wrong');
```

Comment:
Rather than calling this `setStatsText` every time the user adds an input, I would wait until the game is ended to update the stats text or when the user clicks the button. This is adding more work when you are going to keep updating it anyways.